test/model/autocomplete_test.dart

```diff
@@ -274,35 +276,35 @@ void main() {
     }
   });
 
-  test('MentionAutocompleteView mutating store.users while in progress causes retry', () async {
-    const narrow = CombinedFeedNarrow();
+  test('MentionAutocompleteView mutating store.users while in progress does not '
+      'interrupt the current query results', () async {
```

Comment:
nit:
```suggestion
  test('MentionAutocompleteView mutating store.users while in progress does not '
      'prevent query from finishing', () async {
```

"interrupt" reads to me as more about what happens internally — e.g. the old version does get interrupted, even though it then resumes and recovers. The test is really about the ultimate output, which is why the old version passes the test (modulo the detail about checking the new user 11000 is absent).